packages/renderer/src/lib/network/NetworkDetailsSummary.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+import { Link } from '@podman-desktop/ui-svelte';
+
+import { handleNavigation } from '/@/navigation';
+import { NavigationPage } from '/@api/navigation-page';
+
+import DetailsCell from '../details/DetailsCell.svelte';
```

Comment:
nitpick: path alias `import DetailsCell from '/@/lib/details/DetailsCell.svelte';` can be used here